ts/src/lbank.ts

```diff
@@ -830,6 +833,7 @@ export default class lbank extends Exchange {
             const queryInner = this.keysort (this.extend ({
                 'api_key': this.apiKey,
             }, params));
```

Comment:
@ttodua isn't the key here?

Reply:
i made verbose and api-key wasnt being sent, idk i havent looked in other areas, as I have said, it might involve thorough review of whole old api and I think this easy edit fixed the problem (also that user probably migrated into v2 and havent complained again). so, I cant say what was queryInner doing there, what I saw was that private api didnt work at all till date (apikey not being sent) as mentioned here https://github.com/LBank-exchange/lbank-official-api-docs/blob/master/API-For-Spot-EN/Trading%20REST%20API.md#2-place-order